pennylane/plugins/default_qubit.py

```diff
@@ -421,7 +421,7 @@ def _get_operator_matrix(self, operation, par):
         """Get the operator matrix for a given operation or observable.
 
         If the inverse was defined for an operation or observable that requires parameters, returns the
-        operator matrix obtained using negated paramaters.
+        conjugate transpose of the operator matrix.
```

Comment:
This isn't clear. Why would the inverse of an observable be it's conjugate transpose?

Reply:
Thanks, that's true!

The docstring does not reflect clearly, that currently, the inverse is only defined for operations (hence, in this case, the observable is an operation as well).

Modifying it!